src/CodeStyle/CSharp/CodeFixes/Microsoft.CodeAnalysis.CSharp.CodeStyle.Fixes.csproj

```diff
@@ -18,6 +18,10 @@
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_GetFilesToPackage</TargetsForTfmSpecificContentInPackage>
     <!-- Remove once https://github.com/NuGet/Home/issues/8583 is fixed -->
     <NoWarn>$(NoWarn);NU5128</NoWarn>
+
+    <!-- Don't inject a project reference for solution build order dependency on CodeStyleConfigFileGenerator
+         because (when in BuildProjectReferences=false) that would call GetTargetPath on its OUTER build which doesn't have such a target -->
+    <AddSyntheticProjectReferencesForSolutionDependencies>false</AddSyntheticProjectReferencesForSolutionDependencies>
```

Comment:
@rainersigwald Which bug do you want to use to track this workaround? This way we could eventually remove the hack.